package.json

```
@@ -36,13 +35,13 @@
     "test:e2e": "playwright test",
     "format": "biome check",
     "format:fix": "biome check --write",
-    "prepare": "lefthook install"
+    "prepare": "lefthook install",
+    "serve": "bun run ./e2e/index.ts"
```

Lets indicate that this is e2e related
```suggestion
    "serve:e2e": "bun run ./e2e/index.ts"
```